### ISSUE TITLE
add several target classes to visualization

### ISSUE THREAD

**Issue by sennierer** [dasch124/modeller#2]
currently its possible to do the following:
``` xml
  <relation ID="CourtHostsFunction">
      <sourceClass target="function">
          <arity>1</arity>
      </sourceClass>
      <targetClass target="court institution">
          <arity>1</arity>
      </targetClass>
      <name>ist Teil von</name>
      <reverseName>inkludiert</reverseName>
  </relation>
 ```
by separating two target classes with whitespaces the correct relations are created in the lists. However, those relations are not included in the ERM overview chart.
Given that these relations can also be added by just adding the inverse relation it is not urgent to fix that.

**Comment by dasch124** [comment]
should be fixed with c1548dab02

**Comment by sennierer** [comment]
yes, working in the new version, thx
